test/clj_kondo/analysis_test.clj

```diff
@@ -326,6 +326,34 @@
         :to clojure.core}]
      var-usages)))
 
+(deftest ^:test-refresh/focus analysis-signatures-test
```

Comment:
:test-refresh/focus seems to be a tooling-specific thing, can you remove this?

Reply:
this, and the other comments in the issue should be resolved now.